mock/data/soknader.json

```diff
@@ -69,7 +71,12 @@
             "max": "2019-03-21",
             "pavirkerAndreSporsmal": false,
             "kriterieForVisningAvUndersporsmal": null,
-            "svar": [],
+              "svar": [
+                  {
+                      "verdi": "{\"fom\": \"2019-06-25\", \"tom\": \"2019-06-25\"}",
+                      "avgittAv": "TIDLIGERE_SOKNAD"
```

Comment:
Er dette et nytt konsept innført i syfosoknad?

Reply:
Ja. Er for å kunne vite om verdien er preutfylt eller ikke. Kom gjerne med forslag til annen navn/verdi. Var ikke helt fornøyd, men kom ikke på noe bedre

Reply:
Jeg tenker det forsåvidt er fair navn. Men det blir litt rart når det er flyttet opp på spørsmålet.. 🤔 

Da er det kanskje riktigere med `preutfylt: true | false`?